### PR TITLE
Release/2.5.3

### PR DESCRIPTION
**Pull request by KaneCheshire**

### CODE REVIEW

File: Example/Pods/Pods.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -1141,7 +1141,7 @@
 				GCC_PREFIX_HEADER = "Target Support Files/ShowTime/ShowTime-prefix.pch";
 				INFOPLIST_FILE = "Target Support Files/ShowTime/ShowTime-Info.plist";
 				INSTALL_PATH = "$(LOCAL_LIBRARY_DIR)/Frameworks";
-				IPHONEOS_DEPLOYMENT_TARGET = 8.2;
+				IPHONEOS_DEPLOYMENT_TARGET = 12.0;
```
Comment (stuffmc) [inline Example/Pods/Pods.xcodeproj/project.pbxproj:1144]: I think it's good that you support `12.0` here, but then why do the `Package.swift` supports `9`? IMHO the Package could support 12 as well, so you're good to go for the foreseeable years :) The real question is: If you say `12` here, the Package would crash on `9`, right?

**Review by stuffmc — COMMENTED** [review]
I'd obviously prefer if someone else reviews that, but the most important thing is that it seems the SPM supports iOS 9 whereas the library itself supports iOS 12. There's very little reason to support earlier than 12 those days. It means supporting devices older than iPad Air and iPhone 5S.

**Comment by KaneCheshire** [comment]
I'm alright dropping support entirely for 8 :) I'll just update the Package file. I think the worst that will happen is it just won't compile for 8 but if anyone is still supporting it they can just use the previous version. Good catch though!

**Comment by KaneCheshire** [comment]
Sorry I misread that at first. I think when I first created ShowTime it was back when iOS 8 was still around and Cocoapoads was the best way to install small dependencies, and then recently added support for SPM which I imagine is only going to be used for newer projects which are unlikely to be supporting that far back. 

I don't mind just supporting as far back as each solution allows without having any weird warnings

File: Example/Pods/Pods.xcodeproj/project.pbxproj (hunk 1 above)
Comment (stuffmc) [reply Example/Pods/Pods.xcodeproj/project.pbxproj:1144]: @KaneCheshire It's potentially an option to support iOS 9 in Xcode as well. Unsure if you'll get a warning. You decide :) The problem that I see for you only supporting iOS 9 is that I'm pretty sure Xcode 13 will not support anything older than iOS 12, by no means iOS 9.

Comment (KaneCheshire) [reply Example/Pods/Pods.xcodeproj/project.pbxproj:1144]: This is just the value set in the example project, not the lib itself. The `podspec` defines the minimum target so this is fine, it just means that the example project supports iOS 12

**Pull request merged**